src/js/blight/BlightContainer.jsx

```diff
@@ -1,10 +1,41 @@
 import React, { Component } from "react";
 
 export default class BlightContainer extends Component {
+  constructor(props) {
+    super(props);
+    this.totalBlight = this.props.players * 2;
+
+    this.state = { value: 10 };
+
+    this.handleClickP = this.handleClickP.bind(this);
+    this.handleClickM = this.handleClickM.bind(this);
+    this.handleChange = this.handleChange.bind(this);
+  }
+  handleClickP(e) {
+    const newBlight = this.state.value++;
```

Comment:
`this.state.value++` will actually "try" to modify `this.state.value`, and you'd get the warning in your console:
`Line 15:  Do not mutate state directly. Use setState()  react/no-direct-mutation-state`

you can just do something like
`const blight = this.state.value + 1`
or even "better", all in one line
`this.setState({ this.state.value + 1 })`